klobalx/src/main/java/com/mobillium/klobalx/nativeext/FloatExt.kt

```diff
@@ -0,0 +1,3 @@
+package com.mobillium.klobalx.nativeext
+
+fun Float?.orZero() = this ?: false
```

Comment:
🥚

Reply:
val Float?.orZero: Int
    get() = this ?: 0

olarak değişecek

Reply:
val Float?.orZero: Int
get() = this ?: 0

olarak değişecek